var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -76,7 +77,7 @@ class Cuda(Package):
     # https://www.nvidia.com/en-us/drivers/cuda/mac-driver-archive/ mention
     # Mojave support -- only macOS High Sierra 10.13 is supported.
     conflicts('arch=darwin-mojave-x86_64')
-
+    depends_on('gcc@:9.3.2') 
```

Comment:
We handle compiler conflicts in the cuda mixin class (in lib/), please do not add here.

Reply:
But without that cuda can get built with gcc@10.1 or higher.

Reply:
I mean, cube can be built presumably whatever the recent GCC but it causes a run time error.

Reply:
the cuda package does not get "built" in the common sense, it just extracts things. conflicts need to be handled for dependent packages that use the Spack `cuda` package with a selected `%hostCompiler`.

Reply:
```suggestion

```